src/cai_lab_to_nwb/zaki_2024/behaviorinterface.py

```diff
@@ -33,6 +37,20 @@ def add_to_nwbfile(self, nwbfile: NWBFile, metadata: Optional[dict] = None):
 
         freezing_behavior_df = pd.read_csv(self.file_path)
 
+        #Extract motion data
+        motion_data = freezing_behavior_df["Motion"].values
+
+        motion_series = TimeSeries(
+            name="MotionSeries",
+            description="ezTrack measures the motion of the animal by assessing the number of pixels of the behavioral "
+                        "video whose grayscale change exceeds a particular threshold from one frame to the next.",
+            data=motion_data,
+            unit="n.a",
+            starting_time=freezing_behavior_df["Frame"][0] / self.video_sampling_frequency,
```

Comment:
We should add this to a TODO: for checking when we do the synch.

Reply:
added here https://github.com/orgs/catalystneuro/projects/3/views/15?pane=issue&itemId=76549171&issue=catalystneuro%7Ccai-lab-to-nwb%7C1#:~:text=eztrack%20motion%20series